src/app/components/admin/page/statistics-page/statistic-table/statistic-table.component.html

```
@@ -1,29 +1,61 @@
-    <table mat-table [dataSource]="displayData || []">
-      <ng-container matColumnDef="camperPlaceNumber">
-        <th mat-header-cell *matHeaderCellDef> Nr. parceli</th>
-        <td mat-cell *matCellDef="let displayData"> {{ displayData.camperPlaceNumber }}</td>
-        <td mat-footer-cell *matFooterCellDef>Suma:</td>
-      </ng-container>
-      <ng-container matColumnDef="reservationCount">
-        <th mat-header-cell *matHeaderCellDef> Ilość rezerwacji</th>
-        <td mat-cell *matCellDef="let displayData"> {{ displayData.reservationCount }}</td>
-        <td mat-footer-cell *matFooterCellDef>{{ countResCount() }}</td>
-      </ng-container>
+<link rel="stylesheet" href="../../../popups/search-by-popup/search-by-popup.component.css">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)